apps/site/src/components/pages/home/composable-interfaces.tsx

```diff
@@ -1,81 +1,460 @@
-import { Box, Typography } from "@mui/material";
+import { faFigma } from "@fortawesome/free-brands-svg-icons";
+import { Box, Container, Stack, Tooltip, Typography } from "@mui/material";
 import Image from "next/legacy/image";
+import { ReactNode } from "react";
 
-import composableFullImage from "../../../../public/assets/new-home/composable-full-min.png";
+import composableFullImage from "../../../../public/assets/new-home/composable-full-min.webp";
+import {
+  FontAwesomeIcon,
+  GithubIcon,
+  HashIcon,
+  WordPressIcon,
+} from "../../icons";
+import { ArrowUpRightFromSquareIcon } from "../../icons/arrow-up-right-from-square-icon";
+import { BoxesStackedIcon } from "../../icons/boxes-stacked-icon";
+import { BrowserIcon } from "../../icons/browser-icon";
+import { Grid2PlusIcon } from "../../icons/grid-2-plus";
+import { SparklesIcon } from "../../icons/sparkles-icon";
+import { Link } from "../../link";
+import { LinkButton } from "../../link-button";
 
-export const ComposableInterfaces = () => {
+const IconButtonWithTooltip = ({
+  href,
+  label,
+  icon,
+}: {
+  href: string;
+  label: string;
+  icon: ReactNode;
+}) => {
   return (
-    <Box
-      sx={{
-        background:
-          "radial-gradient(116.02% 95.04% at 50% 100.79%, #F3F0F9 0%, #FFFFFF 70.54%);",
-        mt: 2,
-        pb: { xs: 5, sm: 8 },
-        borderBottom: `1px solid #eceaf1`,
-        overflow: "hidden",
+    <Tooltip
+      componentsProps={{
+        tooltip: {
+          // eslint-disable-next-line no-useless-computed-key
+          ["data-testid"]: `${label}-tooltip`,
+        } as any,
       }}
+      title={
+        <Box display="flex" alignItems="center">
+          {label}
+          <ArrowUpRightFromSquareIcon
+            sx={{
+              fontSize: 12,
+              fill: ({ palette }) => palette.gray[50],
+              ml: 1.25,
+            }}
+          />
+        </Box>
+      }
+      placement="top"
     >
+      <LinkButton
+        linkProps={{
+          // eslint-disable-next-line no-useless-computed-key
+          ["data-testid"]: `${label}-button`,
+        }}
+        data-testid={label}
+        href={href}
+        color="inherit"
+        sx={{
+          borderRadius: 2.5,
+          background: "transparent",
+          p: 0.5,
+          minWidth: 0,
+          "&:hover": {
+            background: ({ palette }) => palette.gray[20],
+          },
+        }}
+      >
+        {icon}
+      </LinkButton>
+    </Tooltip>
+  );
+};
+
+export const ComposableInterfaces = () => {
```

Comment:
The name of this component no longer makes sense – it's more `SupportedApplications` or something like that now

Reply:
I was overcomplicating this, thank you. Fixed and changed the component name to your suggestion in 96380cfcfc0a99ee124ea742150902fe85128e61